src/Rector/Contrib/PHPUnit/SpecificMethodRector.php

```diff
@@ -24,7 +24,7 @@
     /**
      * @var string[][]|false[][]
      */
-    private $oldToNewMethods = [
+    private $defaultOldToNewMethods = [
```

Comment:
Maybe `$renameMethodsMap`? As we are "mapping"

Reply:
That would be intent, a verb, which suggest action suitable for method.
This is explicit name map with both values description.